python/llm/dev/benchmark/all-in-one/run.py

```diff
@@ -432,12 +444,39 @@ def run_transformer_int4_gpu(repo_id,
             thread = threading.Thread(target=run_model_in_thread, args=(model, in_out, tokenizer, result, warm_up, num_beams, input_ids, out_len, actual_in_len, num_trials))
             thread.start()
             thread.join()
+
+            if result[in_out]:
+                new_row = []
+                first_token_latency = round(np.mean(result[in_out], axis=0)[0]*1000.0, 2)
+                rest_token_latency = round(np.mean(result[in_out], axis=0)[1]*1000.0, 2)
+                encoder_time = round(np.mean(result[in_out], axis=0)[2]*1000.0, 2)
+                input_output_tokens = in_out
+                actual_input_output_tokens = f'{int(np.mean(result[in_out], axis=0)[3])}' + f'-{int(np.mean(result[in_out], axis=0)[4])}'
+                cpu_embedding = cpu_embedding if 'win' in test_api else 'N/A'
```

Comment:
don't need this argument, it is always 'N/A'

Reply:
ok